### ISSUE TITLE
suggestion- not an Issue, but Possibility of creating a PiperTTS extension for SillyTavern?

### ISSUE THREAD

**Issue by LJtrix** [tijo95/win_tts_piper#2]
This would be super awesome, as Silly Tavern has so many cool features like group chats, branches, and much much more.

I was asking if there is a chance you could do it (it's okay if not) 
there is an XTTS extension, it's great, , but the TTS always takes a while to load. While piper  is obviously God Tier fast.

And Since I know how to train voices, it would be fun to make the TTS even more realistic.

So if you do make an extension, it would be awesome to replicate it to how the XTTS extension is, where  for  each character, you can select a voice. (so let's say in a group chat, we have, Batman and Wonderwoman, For each character I can choose a speaker, so batman would be a batman voice, and Wonder woman  would be a wonderwoman voice  while in a group chat. 

If  you do end up making this, this would be awesome! and if not, that is also fine. 

So basically what would be cool is Piper TTS, with with a voice  for each character.

**Comment by FemBoxbrawl** [comment]
I think this would actually be awesome! especially for lower end PCs!
